packages/core/src/common/keybinding.ts

```diff
@@ -144,27 +151,62 @@ export class KeybindingRegistry {
         }
     }
 
-    registerKeybindings(...bindings: Keybinding[]): void {
+    registerKeybindings(...bindings: (RawKeybinding | Keybinding)[]): void {
```

Comment:
It would be nice to rename `RawKeybinding` to `Keybinding` in one of the next PRs.

Reply:
Yes I tought of that but would you have an idea what you call the old Keybinding after ? like InternalKeybinding was long... PrivKeybinding.. I was kinda quite unsure about that one.

Reply:
One idea may be to use UserKeybinding too for RawKeybinding